webapp/src/components/util.ts

```diff
@@ -57,7 +61,24 @@ export function rangeToStr(
   if (rowStart === rowEnd && columnStart === columnEnd) {
     return `${sheetName}${columnNameFromNumber(columnStart)}${rowStart}`;
   }
-  return `${sheetName}${columnNameFromNumber(
-    columnStart,
-  )}${rowStart}:${columnNameFromNumber(columnEnd)}${rowEnd}`;
+  return `${sheetName}${columnNameFromNumber(columnStart)}${rowStart}:${columnNameFromNumber(
+    columnEnd,
+  )}${rowEnd}`;
+}
+
+export function getFullRangeToString(
+  selectedView: SelectedView,
+  worksheets: WorksheetProperties[], // solo pasar names
+): string {
+  // order of values is confusing compared to rangeToStr range type, needs refactoring for consistency
```

Comment:
I've added a unit test and refactored params to use only worksheetNames instead of worksheets.
Regarding my comment in line 73, this should be done later on as many files could be affected